indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -710,8 +736,8 @@ private boolean announceTask(
         statusLock.wait(waitMs);
         long elapsed = timeoutStopwatch.elapsed(TimeUnit.MILLISECONDS);
         if (elapsed >= waitMs) {
-          log.error(
-              "Something went wrong! [%s] never ran task [%s]! Timeout: (%s >= %s)!",
+          log.makeAlert(
+              "Something went wrong with worker [%s], never ran task [%s]! Timeout: (%s >= %s)!",
```

Comment:
Generally alert descriptions should be short & simple, as folks have built systems to consolidate alerts with the same description for emails & analysis. So something like `Task assignment timed out` or `Task assignment timed out for worker[%s]` would be good imo, with the additional information (like elapsed & task id) added to the data with `addData`.

Could also do both an alert and a `log.error`.